straxen/corrections_services.py

```diff
@@ -241,6 +249,24 @@ def get_config_from_cmt(self, run_id, model_type, version='ONLINE'):
                              f"Please contact CMT manager and yell at him")
         return file_name
 
+    def check_for_nans(self, when, df, version):
```

Comment:
Please use pandas builtin functions so the code is more readable. eg: `df.loc[:when,version].isnull().any()`

Reply:
Hi Yossi,
Great suggestion, thanks! 
(I posted a notebook where I test this, see [here](https://github.com/XENONnT/corrections/blob/master/CMT/examples/Check_NaNs.ipynb) )